tox.ini

```diff
@@ -43,6 +43,7 @@ deps =
     -r requirements/test.txt
     -r requirements/prd.txt
 commands=
+    python -m pip install -r requirements/pytorch.cpu.txt
```

Comment:
@ryanhausen I was just looking at this to resolve the same CI issues I'm having, however, let's chat tomorrow about how this actually works - since this "save" seems too late as ``requirements/prd.txt`` should already be installed above in ``deps`` (I think), and this should have already hit the storage limit. 🤷

Reply:
@jamienoss sounds good. I removed the torch installs from `prd.txt`. We can chat about it tomorrow though

Reply:
😆 Yep, that would do it! Sorry, I wasn't taking a proper look and only noticed it as I was looting your code lol